docs/developing-locust.rst

```diff
@@ -88,7 +91,7 @@ The documentation source is in the `docs/ <https://github.com/locustio/locust/tr
 
     .. code-block:: console
 
-        $ poetry install --with docs
+        $ poetry install --no-root --only docs
```

Comment:
Is the `--no-root` necessary?

Reply:
It saves having to install the project itself - it just sticks the dependencies required to build the docs into the `venv` only, which is quicker

Reply:
Sure. I was just thinking the opposite is more rare (developing the docs without developing the project :)

Anyways, never mind :)

Reply:
It's a fair point! There are a fair few dependencies for building the docs - the alternative would be doing either:

`poetry install --with docs` <-- install/build everything (hopefully it's cached if you built it already) but add the docs dependencies

`poetry install --with dev,test,docs` up front ^^^ when originally installing the editable files and dev package, thus making the docs building available generally if you're developing the package

Reply:
I like `poetry install --with docs`

Reply:
Addressed in https://github.com/locustio/locust/pull/2873/commits/7d7bfc673d7b2676d3491d8f8218510d8d1bfa49